javascript/javascript1/week1/house.js

```diff
@@ -0,0 +1,27 @@
+let firstHouseWidth = 8;
```

Comment:
You should make sure to use `const` if you don't plan on re-assigning a variable.